### PR TITLE
Refactor field names in instrument

### PR DESCRIPTION
**Pull request by tnkhanh**

### CODE REVIEW

**Comment by tnkhanh** [comment]
Fixing #43

File: src/llvm/llinstrument.ml [hunk 1]
```diff
@@ -22,16 +22,16 @@ type ann_type =
...
   | Safe
 
-type instr_with_tag =
-  { pos : position;
-    tag : int;
-    ins : instr
+type tagged_ins =
+  { tagx_pos : position;
+    tagx_tag : int;
+    tagx_ins : instr
```
Comment (taquangtrung) [inline src/llvm/llinstrument.ml:28]: use `instr` instead of `ins` (instr is used by Llvm.ml)

Comment (tnkhanh) [reply src/llvm/llinstrument.ml:28]: so the field will be `tagx_instr` ?

Comment (taquangtrung) [reply src/llvm/llinstrument.ml:28]: Yes, use `instr` for all components refer to instructions

**Review by taquangtrung — APPROVED** [review]

**Pull request merged**